Manual build instructions giving error on master: Could not find snappy

When following the build instructions here: https://github.com/steemit/steem/blob/master/doc/building.md on the `master` branch, the `cmake` command is giving an error:

```
CMake Error at /usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:148 (message):
  Could NOT find snappy (missing: SNAPPY_LIBRARIES SNAPPY_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:388 (_FPHSA_FAILURE_MESSAGE)
  libraries/vendor/rocksdb/cmake/modules/Findsnappy.cmake:17 (find_package_handle_standard_args)
  libraries/vendor/rocksdb/CMakeLists.txt:66 (find_package)
```

After resolving that, it gave a similar error for `bzip2`:

```
CMake Error at /usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake: 148 (message):
  Could NOT find bzip2 (missing: BZIP2_LIBRARIES BZIP2_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:388 (_FPHSA_FAILURE_MESSAGE)
  libraries/vendor/rocksdb/cmake/modules/Findbzip2.cmake:17 (find_package_handle_standard_args)
  libraries/vendor/rocksdb/CMakeLists.txt:82 (find_package)
```

--- a/doc/building.md
+++ b/doc/building.md
@@ -48,6 +48,8 @@ will build out of the box without further effort:
         cmake \
         g++ \
         git \
+        libbz2-dev \
+        libsnappy-dev \
         libssl-dev \
         libtool \
         make \
@@ -102,6 +104,8 @@ Here are the required packages:
         cmake3 \
         g++ \
         git \
+        libbz2-dev \
+        libsnappy-dev \
         libssl-dev \
         libtool \
         make \

